pkg/peer/config/peerconfig.go

```diff
@@ -19,6 +19,7 @@ const (
 	sidetreePortKey        = "sidetree.port"
 	sidetreeTLSCertificate = "sidetree.tls.cert.file"
 	sidetreeTLSKey         = "sidetree.tls.key.file"
+	sidetreeAPIToken       = "sidetree.api.token"
```

Comment:
Will this be a single token for all endpoints?

Reply:
@troyronda ^^^^

Reply:
I think so

Reply:
I think it would be reasonable to have:

- sidetree api read token
- sidetree api read/write token
- system/subcomponent (cas/blockchain/...) api read token
- system/subcomponent (cas/blockchain/...) api read/write token